### DISCUSSION TITLE
Custom init pattern

### DISCUSSION THREAD

**Discussion by darenager** [bangcorrupt/hacktribe#152] [General]
I notice this is mentioned as a feature, how is it changed?

**Comment by bangcorrupt** [comment]
At the moment it needs changing in the hacked firmware, before installing.

See [Set custom init pattern](https://github.com/bangcorrupt/hacktribe/wiki/How-To#set-custom-init-pattern) in the wiki.  

At some point I'll make it so we can just 'Save As Init Pattern' on the device.

**Reply by darenager** [reply]
Ah, thanks, I got the file from someone else, it would be great to have save as init pattern option in the future.
